joycond-cemuhook.py

```diff
@@ -743,6 +737,7 @@ def print_slots():
         if active_devices != taken_slots():
             print_slots()
 
+        loop.run()
```

Comment:
where to put this loop run?
It blocks the other loop...

Reply:
Was looking for asyncio_glib thanks to https://blogs.gnome.org/jamesh/2019/08/05/glib-integration-for-python-asyncio/
But it loop has to be declared in main thread.
It breaks adding a loop in each worker thread...

How to deal with this?

Reply:
we might need to use https://github.com/altdesktop/python-dbus-next
which support asyncio with dbus

Reply:
ok using a simple thread makes it work...